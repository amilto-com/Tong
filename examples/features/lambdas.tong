// Lambda syntaxes

let inc1 = |x| x + 1
let inc2 = \x -> x + 1
print(inc1(10))
print(inc2(10))
