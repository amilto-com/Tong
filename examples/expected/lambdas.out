11 11 19